Insurance_company/insurance_app/models.py

```diff
@@ -46,6 +55,28 @@ def __str__(self):
         return f"Nazwa polisy: {self.policy_name}"
 
 
+class HouseInsurance(models.Model):
+
+    HOUSE_TYPES = (
+        (1, "Dom"),
+        (2, "Szeregowiec"),
+        (3, "Mieszkanie")
+    )
+
+    # predefined fields
+    policy_id = models.CharField(generate_id(), primary_key=True)
+    policy_name = models.CharField(max_length=100)
+    policy_description = models.TextField(max_length=2000)
+    valid_to = models.DateField(auto_now=True)
```

Comment:
czemu przy utworzeniu/modyfikowaniu wpisu valid to od razu będzie w przeszłości, to ma sens?